src/repositories/linked-accounts/schemas/linked-account.schema.ts

```diff
@@ -5,8 +5,8 @@ export type LinkedAccountDocument = HydratedDocument<LinkedAccount>;
 
 @Schema()
 export class LinkedAccount {
-  @Prop({ default: false })
-  verified: boolean;
+  @Prop({ required: true, default: 'unferified', enum: ['verified', 'unverified'] })
+  status: string;
```

Comment:
Typo and better type def

```suggestion
  @Prop({ required: true, default: 'unverified', enum: ['verified', 'unverified'] })
  status: 'verified' | 'unverified';
```

Also, isn't default unused?
```suggestion
  @Prop({ required: true, enum: ['verified', 'unverified'] })
  status: 'verified' | 'unverified';
```